src/ledger/LedgerManager.h

```diff
@@ -51,7 +51,7 @@ class LedgerManager
         LM_BOOTING_STATE,
 
         // local state is in sync with view of consensus coming from herder
-        // desynchronization will cause transition to CATCHING_UP_STATE.
+        // desynchronization will cause transition to LM_BOOTING_STATE.
```

Comment:
Was this just a typo in the previous comment? It looks like you did not change state-transition logic.